src/form/types/types.ts

```diff
@@ -142,6 +142,10 @@ export const NID_VERIFICATION_BUTTON = 'NID_VERIFICATION_BUTTON'
 export const DIVIDER = 'DIVIDER'
 export const HEADING3 = 'HEADING3'
 export const SIGNATURE = 'SIGNATURE'
+export const LINK_BUTTON = 'LINK_BUTTON'
+export const ID_READER = 'ID_READER'
+export const HTTP = 'HTTP'
+export const ID_VERIFICATION_BANNER = 'ID_VERIFICATION_BANNER'
```

Comment:
@tahmidrahman-dsi dont we also need HIDDEN ?

Reply:
Ah yes, we definitely need, on it right away